collect_app/build.gradle

```diff
@@ -286,11 +286,14 @@ dependencies {
     implementation Dependencies.rarepebble_colorpicker
     implementation Dependencies.commons_io
     implementation Dependencies.opencsv
-    implementation(Dependencies.javarosa) {
-        exclude group: 'joda-time'
-        exclude group: 'org.slf4j'
-        exclude group: 'org.hamcrest', module: 'hamcrest-all'
-    }
+    implementation 'net.sf.kxml:kxml2:2.3.0'
+    implementation 'org.apache.commons:commons-csv:1.4'
+    implementation files('javarosa-3.2.0.jar')
```

Comment:
This is still 3.2.0 because it's off the unmerged https://github.com/getodk/javarosa/pull/646 but it's the release candidate for 3.3, basically.